### PR TITLE
Esdoc/NVM

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR add esdoc and nvm support

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v4.2.1
```
Comment (davidbgk) [inline .nvmrc]: Warning, some lib (at least `swifter`) are not compiling on the latest OSX with that version of nodejs. Still investigating.

Comment (noirbizarre) [inline .nvmrc]: OK, I just rebased and rebuilt https://github.com/noirbizarre/sifter.js/tree/nested-properties-built until brianreavis/sifter.js#27 is merged. It should work on Mac too.

Comment (davidbgk) [inline .nvmrc]: @vinyll can you try?

**Comment by vinyll** [comment]
`npm install`ed successfully! :)

**Comment by noirbizarre** [comment]
Great ! :)

**Pull request merged**